packages/react/src/Components/InfoTabs.jsx

```diff
@@ -0,0 +1,61 @@
+// Global imports
+import React, { useState } from 'react';
+
+// Local imports
+import MRZPanel from './MRZPanel';
+import ImagePanel from './ImagePanel';
+
+const InfoTabs = () => {
+
+  const [stateTab, setStateTab] = useState({
+    imagePanel: true,
+    MRZPanel: false,
+  });
+
+  const clickedTab = (panel) => {
+    return panel ? "govuk-tabs__list-item govuk-tabs__list-item--selected" : "govuk-tabs__list-item";
+  }
+
+  return (
+		<div className="govuk-tabs" data-module="govuk-tabs">
+      <ul className="govuk-tabs__list" role="tablist">
+        <li
+          className={ clickedTab(stateTab.imagePanel) }
```

Comment:
This 👇 or something similar might be cleaner.

```suggestion
          className={`
            govuk-tabs__list-item 
            ${ active.imagePanel ? 'govuk-tabs__list-item--selected' : ''}
          `}
```